app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/TasksRepositoryTest.kt

```diff
@@ -0,0 +1,945 @@
+        }
+        totalTestCommonTasks.sortBy { it.title }
+        listCommonTasks.forEachIndexed { index, data ->
+            assertEquals(
+                expected = totalTestCommonTasks[index].title,
+                actual = data.title
+            )
+            assertEquals(
+                expected = totalTestCommonTasks[index].assignee?.fullName,
+                actual = data.assignee?.fullName
+            )
+            assertEquals(
+                expected = totalTestCommonTasks[index].isClosed,
+                actual = data.isClosed
+            )
+        }
+    }
+
+    @Test
+    fun `test get epics`() = runBlocking {
+        val epics = tasksRepository.getEpics(1, FiltersData()).sortedBy { it.title }
+        val testEpics = TestData.projects[0].epics.sortedBy { it.title }
+        val epicsWithFilter = tasksRepository.getEpics(
+            page = 1,
+            filters = FiltersData(
+                createdBy = listOf(
+                    UsersFilter(
+                        id = 0,
+                        name = activeUser.user.fullName,
+                        count = 0
+                    )
+                )
+            )
+        ).sortedBy { it.title }
+        val testEpicsWithFilter = TestData.projects[0].epics
+            .filter { it.creator.fullName == activeUser.user.fullName }
+            .sortedBy { it.title }
+
+        assertEquals(
+            expected = epics.size,
+            actual = testEpics.size
+        )
+        testEpics.forEachIndexed() { index, testEpic ->
+            assertEquals(
+                expected = testEpic.title,
+                actual = epics[index].title
+            )
+            assertEquals(
+                expected = testEpic.assignedTo?.fullName,
+                actual = epics[index].assignee?.fullName
+            )
+            assertEquals(
+                expected = testEpic.isClosed,
+                actual = epics[index].isClosed
+            )
+        }
+
+        assertEquals(
+            expected = epicsWithFilter.size,
+            actual = testEpicsWithFilter.size
+        )
+        testEpicsWithFilter.forEachIndexed() { index, testEpic ->
+            assertEquals(
+                expected = testEpic.title,
+                actual = epicsWithFilter[index].title
+            )
+            assertEquals(
+                expected = testEpic.assignedTo?.fullName,
+                actual = epicsWithFilter[index].assignee?.fullName
+            )
+            assertEquals(
+                expected = testEpic.isClosed,
+                actual = epicsWithFilter[index].isClosed
+            )
+        }
+    }
+
+    @Test
+    fun `test get all user stories`() = runBlocking {
+        val userStories = tasksRepository.getAllUserStories().sortedBy { it.title }
+        val testUsersStory = TestData.projects[0].userstories.sortedBy { it.title }
+
+        userStories.forEachIndexed { index, story ->
```

Comment:
Why not use the same way to assert equality (with sizes assert and stuff) as in the other places. Especially if there were separate function to do that... (you know what I mean)